salt/fileserver/gitfs.py

```diff
@@ -50,7 +50,7 @@
 
 PER_REMOTE_OVERRIDES = ('base', 'mountpoint', 'root', 'ssl_verify',
                         'env_whitelist', 'env_blacklist', 'refspecs')
-PER_REMOTE_ONLY = ('name', 'saltenv')
+PER_REMOTE_ONLY = ('formula', 'name', 'saltenv')
```

Comment:
@terminalmage my bad! I forgot to mention when I typed up the original PR, I tried global, but it kept throwing a syntax error, seems to be a reserved keyword.